packages/filebrowser/src/browser.ts

```diff
@@ -179,6 +179,28 @@ export class FileBrowser extends SidePanel {
     }
   }
 
+  /**
+   * Number of directory items to show on the left side of the ellipsis in breadcrumbs.
+   */
+  get minimumBreadcrumbsLeftItems(): number {
+    return this.crumbs.minimumLeftItems;
+  }
+
+  set minimumBreadcrumbsLeftItems(value: number) {
+    this.crumbs.minimumLeftItems = value;
+  }
```

Comment:
Ah we need to do this because `crumbs` are protected. I guess it's fine to add this getters and setters then.